pkg/controller/installation/core_controller.go

```diff
@@ -285,29 +285,40 @@ func fillDefaults(instance *operator.Installation) error {
 			if v4pool.NodeSelector == "" {
 				v4pool.NodeSelector = operator.NodeSelectorDefault
 			}
-			// Default IPv4 address detection to "first found" if not specified.
 			if instance.Spec.CalicoNetwork.NodeAddressAutodetectionV4 == nil {
+				// Default IPv4 address detection to "first found" if not specified.
 				t := true
 				instance.Spec.CalicoNetwork.NodeAddressAutodetectionV4 = &operator.NodeAddressAutodetection{
 					FirstFound: &t,
 				}
 			}
+			if v4pool.BlockSize == nil {
+				var twentySix int32 = 26
```

Comment:
The variable name feels weird to me. (not requesting you change it, just a comment)